Gemfile

```diff
@@ -1,7 +1,7 @@
 source ENV['GEM_SOURCE'] || "https://rubygems.org"
 
 group :development, :unit_tests do
-  gem 'rake',                                              :require => false
+  gem 'rake', '< 11.0.1',                                  :require => false
```

Comment:
OK. I'll actually update that as we use modulesync and this change is to be applied.